docs/differential-analysis.rst

```diff
@@ -0,0 +1,165 @@
+         'newer_run_create_time': '2022-08-10T13:07:44.194Z',
+         'newer_run_id': 'jcdqsju4utpaayj5dh5r2llzffeolg0u',
+         'newer_run_not_responded_devices': [],
+         'older_run_create_time': '2022-08-10T12:57:03.872Z',
+         'older_run_id': 'yhbg3wcea9y1l4asiltky5tupkgauzas',
+         'older_run_not_responded_devices': []}
+
+
+In the example response you can see that ``AdBlocker Ultimate`` add-on was added between the two snapshot intervals.
+
+
+Filter Devices
+##############
+
+Using the ``.set_device_ids()`` you can narrow down the query to a specific devices only. The method accepts an array of integers.
+To use this method, append it to the rest of the Differential object query or combine it with any of the other methods.
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.audit_remediation import Differential
+    >>>
+    >>> cbc = CBCloudAPI(profile='sample')
+    >>>
+    >>> resp = cbc.select(Differential).newer_run_id('jcdqsju4utpaayj5dh5r2llzffeolg0u').older_run_id('yhbg3wcea9y1l4asiltky5tupkgauzas')
+    >>> actual_changes = resp.count_only(False).set_device_ids([12345])
+    >>> print(actual_changes[0]._info)
+        {'diff_processed_time': 0.039,
+         'diff_results': [{'added_count': 1,
+                           'change_count': 1,
+                           'changes': [{'action': 'ADDED',
+                                        'fields': [{'key': 'name',
+                                                    'value': 'AdBlocker Ultimate'}]}],
+                           'device_id': 12345,
+                           'newer_run_row_count': 21,
+                           'older_run_row_count': 20,
+                           'removed_count': 0}],
+         'newer_run_create_time': '2022-08-10T13:07:44.194Z',
+         'newer_run_id': 'jcdqsju4utpaayj5dh5r2llzffeolg0u',
+         'newer_run_not_responded_devices': [],
+         'older_run_create_time': '2022-08-10T12:57:03.872Z',
+         'older_run_id': 'yhbg3wcea9y1l4asiltky5tupkgauzas',
+         'older_run_not_responded_devices': []}
+
+
+
+Export Results
+##############
+
+Using the ``.async_export()`` you can create an asynchronous job that exports the results from the run.
+To use this method, append it to the rest of the Differential object query or combine it with any of the other methods.
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.audit_remediation import Differential
+    >>>
+    >>> cbc = CBCloudAPI(profile='sample')
+    >>>
+    >>> resp = cbc.select(Differential).newer_run_id('jcdqsju4utpaayj5dh5r2llzffeolg0u').older_run_id('yhbg3wcea9y1l4asiltky5tupkgauzas')
+    >>> export = resp.count_only(False).set_device_ids([12345]).async_export()
+    >>> print(export.status)
+    IN_PROGRESS
+    >>> # wait for it to finish and refresh the information in the SDK
+    >>> job_future = export.await_completion()
+    >>> finished_job = job_future.result()
+    >>> finished_job.refresh()
```

Comment:
Why do we need to refresh the finished_job as we did `await_completion()` and `result()`?

Reply:
Something went very wrong here, most of the stuff is not needed, thanks!